### DISCUSSION TITLE
Logging wrong CQ-Zone in US/CANADA?

### DISCUSSION THREAD

**Discussion by dk5cr** [magicbug/Cloudlog#2902] [General]
By showing my confirmed QSOs cloudlog showed me a missing CQ-Zone 3 although i have a QSL-checked QSO with OMs from California, Washington and Oregon. 
With SQL-tools it showed the CQ-zones for US and Canada and table TABLE_HRD_CONTACTS_V01 showed me only COL_CQZ-entries with 4 (Canada) and 5 (US+Canada). 
Unfortunately the "real" CQZone isn't stored in the QSO-entry. 

My version is 2.6.1. 
DXCC Tables a.s.o. are up to date. 
Cloudlog is running on raspi4 without other errors. 

I would appreciate to know if there's a failure in logging als US-QSO to CQZ 4 and CANADA-QSO to CQZ 4 and 5?! 

Thanx and Happy New Year 
Cornel / DK5CR

BTW: Is there a chance to correct the CQ-zone by updating via SQL on COL_GRIDZONES?
BTW2: Searching for GRID and CQZ doesn't give me matching results to my problem, so if there is a solution, please let me know the discussion-number Thx.

**Comment by magicbug** [comment]
We do now grab the CQ zone from callbooks if its provided, but beyond that there is no way of knowing what Zone someone is in so we default to that of the DXCC that we get from Clublogs data files, you can of course edit a QSO and correct the zone.
